### PR TITLE
L1-289: Api inflation

### PR DESCRIPTION
**Pull request by kostekIV**

### CODE REVIEW

**Comment by Marcin-Radecki** [comment]
Let's merge https://github.com/Cardinal-Cryptography/aleph-node/pull/1816 first, so we can review this PR properly

**Review by Marcin-Radecki — APPROVED** [review]

File: primitives/src/lib.rs [hunk 1]
```diff
@@ -288,6 +288,10 @@ sp_api::decl_runtime_apis! {
         /// also as `aleph_key` - consensus engine's part of session keys) in the current session
         /// of AlephBFT (finalisation committee).
         fn key_owner(key: AuthorityId) -> Option<AccountId>;
+        /// Returns inflation from now to now + 1 year.
```
Comment (lesniak43) [inline primitives/src/lib.rs:291]: Add "capped at 100%" or something like that - `Perbill` can only represent numbers not larger than one.

**Review by lesniak43 — APPROVED** [review]
LGTM

File: primitives/src/lib.rs (hunk 1 above)
Comment (kostekIV) [reply primitives/src/lib.rs]: done

**Pull request merged**